scripts/benchmarks/benches/compute.py

```diff
@@ -60,7 +60,7 @@ def bin_args(self) -> list[str]:
         return []
 
     def extra_env_vars(self) -> dict:
-        return {}
+        return {"UR_ADAPTERS_FORCE_LOAD" : os.path.join(self.bench.libs, f"libur_adapter_{options.ur_adapter_name}.so")}
```

Comment:
hm, maybe we should move this to base benchmark and concatenate the returned values?

Reply:
Yeah, we can do that. Done.